chia/cmds/chia.py

```diff
@@ -149,7 +133,6 @@ def run_daemon_cmd(ctx: click.Context, wait_for_unlock: bool) -> None:
 
 
 def main() -> None:
-    monkey_patch_click()
```

Comment:
This monkeypatch was added as having a non standard locale meant that a large minority of macos users could not plot using the GUI as the command would fail when passed from the GUI. Are we comfortable that that is no longer an issue?

Reply:
To shortcut a few links, https://github.com/pallets/click/commit/9072f1809e559898814d58da77b9b9d5a73233fb is the commit that entirely deletes the function we monkey patch.

Reply:
Them dropping locale certainly seems like it would help. On the testing side @richardkiss would have been the test writer on this if one was written.

Reply:
I was more curious what group of users would be affected by this in case we have one in house that could recreate the issues with click 7 without monkey patching and then see if this pr works in that same situation.

Reply:
I manually tested this build on Intel with macOS 12.5.1 (Monterey), macOS 11.6.4 (Big Sur), macOS 10.15.7 (Catalina) and macOS 10.14.6 (Mojave) - all worked fine running and making a plot from the GUI. I'll note a caveat that I only testing the English/North America locale/region setting.

Reply:
From source?  Or via the dmgs.

Reply:
DMGs from this PR

Reply:
@emlowe Were you able to reproduce the problem with Click 7?

Reply:
I didn't try that but I suppose I could - but that would require a build

Reply:
I could not reproduce any problems with a build that just removes the monkey path and uses click 7 (tested on Monterey and Mojave).

Reply:
I'm not sure what combination of python/pyinstaller/click were involved in the original need to monkey-patch click. It seems that possibly hasn't been needed due to other changes elsewhere (pyinstaller or the build systems?). In any event, I think we should move forward with this PR. To that end, I am resolving this and undrafting.